### DISCUSSION TITLE
suggestion more than an issue

### DISCUSSION THREAD

**Discussion by Reality4DEvolution** [5axes/TabAntiWarping#9] [Ideas]
the anti warping idea is great.  but, problem with all stuff in early layers is removal from bed.
in sketchup, i actually made little attachable tabs to put on  my other projects, they had a reverse bevel so that the distal corner was free from the bed, could get the scraper under it, and start to remove the project much easier.
if you combined such an effect, into the anti warping circles, it would be fantastic !
anyway, thanks !
![509](https://user-images.githubusercontent.com/67389333/108363197-a6be1000-71c2-11eb-9ea5-2ac56a9e2fbd.jpg)

**Comment by 5axes** [comment]
Hello,
Nice Idea, but not so easy because the bevel should be created out of the part. And it's not an obvious modification to the current plugin

**Comment by Reality4DEvolution** [comment]
thanks for reply.
no problem.  but, the bevel and that tab are actualy additions.  it is a small 2mm x 10 mm piece, i just attach it to an edge of what i am making.  then, when done, that bevel (long part on top) can be lifted, with the scraper, to start the removal process (and it also helped to hold the item down).  then, when done, it c an be sliced off.

re addiding it to current plugin, the plugin (circle) could be designed to be about 1 mm high, and be bigger in diameter at thte top than at the botom, thus creating a bevel.

but, again, i understand the design complexities, etc.

and thanks for the plugin !

**Comment by 5axes** [comment]
Like a capsule ?
![image](https://user-images.githubusercontent.com/11015345/108533436-42da3b00-72d9-11eb-8a5b-97d3eb7a82ed.png)

![image](https://user-images.githubusercontent.com/11015345/108533482-508fc080-72d9-11eb-8160-91e9ea0e2d31.png)

**Comment by 5axes** [comment]
Need some tests to find the best height  ( Don't want to add more parameter )

![image](https://user-images.githubusercontent.com/11015345/108536021-3d322480-72dc-11eb-932c-1b0731d655ef.png)

[https://github.com/5axes/TabAntiWarping/releases/tag/V1.1.0](https://github.com/5axes/TabAntiWarping/releases/tag/V1.1.0)

**Comment by Reality4DEvolution** [comment]
your deslgn looks good !
question would be that if the plate/ring is not FILLED (end itself just bevels up), if the plate/ring would have enough strength to be able to take the scraper underneath it.  here is an example of what i did with thse wings.
but you got the idea !  and if that stayed strong enough, enough to get teh scraper blade underneath it and just under the actual model printed, it would work !
![515](https://user-images.githubusercontent.com/67389333/108551045-57a8d580-72bd-11eb-8c1f-15b2efd54c69.jpg)

**Comment by 5axes** [comment]
Seems to be ok, 
![20210219_221815](https://user-images.githubusercontent.com/11015345/108562850-c78c7f80-7300-11eb-97d9-4dfdeddc8bbc.jpg)

**Comment by 5axes** [comment]
![20210219_221703](https://user-images.githubusercontent.com/11015345/108563192-59948800-7301-11eb-853c-8d1a6d076513.jpg)

**Comment by Reality4DEvolution** [comment]
nice !  I will download and test later (printing several things for toolshop right now )
thans for looking at this !

**Comment by Reality4DEvolution** [comment]
ok, was testing it, i hve ver 1.0.1, cant find update?  went to marketplace, 1.0.1 was what what got installed as new install??
thanks
 !
nick
NOTE, ADENDUM, IGNORE this, i just figured out how to download form github (am just learning), got the update, will test, thakns

**Comment by Reality4DEvolution** [comment]
now THAT is absolutely COOL !!!  
i printed a 10mm x 2mm block, just to test, and added a 5mm ring to it.
the TAPER is great !  scraper just fits under it, and if i make ring bigger, it will give more support to actual object printed, allowing scraper to get under it and start removing from bed !
thanks very much for looking into this, AND for your plug-in

**Comment by Reality4DEvolution** [comment]
addendum -
i tested it, at 0.16mm, and 5 mm circle.
problem was that with putting the scraper underneath, it broke off the support, could not get under the object.
i will test today later, with larger diameter.
question, what does the x/y value do, and is there a way to make it thicker (more layers in the body?
again, thanks !

**Comment by 5axes** [comment]
X/Y = distance between the support and the part
The goal of this plugin is just to create a geometry, the result depends of the Cura possibilities to create the support, and unfortunatly it's not so easy to create a support base  thicker than just one layer. It's not impossible but it really depends of your settings

**Comment by Reality4DEvolution** [comment]
no problem at all, i will experiment, and again i just appreciate your experimenting with this !
i just created a new project, added this to it, left it at x/y of zero, and will see how that does.
thanks again !

**Comment by Reality4DEvolution** [comment]
it WORKS !  using an x/y value of zero, it adheres to the printed object better, and I WAS ABLE to get the scraper underneath the beveled edge, Underneath the TAB, and then UNDERNEATH the object itself, without BREAKING the TAB.  that made it much easier to remove the object.
I believe you have made the best of both worlds -  an ADHERING tab, and a REMOVING tab.

ADDENDUM, 2021-02-26, 4 pm -  did well on flat surface again !  keep at ZERO for x/y, and i used 10 mm diameter.  it was able to lift the bevel, / edge, and get under the edge of the actual object.
